examples/fabric-sync/bridge/src/Bridge.cpp

```diff
@@ -45,6 +45,83 @@ namespace bridge {
 
 namespace {
 
+class AdministratorCommissioningCommandHandler : public CommandHandlerInterface
```

Comment:
Where are you instantiating an instance of this class and registering it to actually do something?